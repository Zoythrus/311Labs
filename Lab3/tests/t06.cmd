 text0.txt 4
