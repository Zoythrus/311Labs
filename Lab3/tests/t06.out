0 1 8 3 10 5 7 4 2 9 6 
